site/layouts/shortcodes/orange-supra.html

```diff
@@ -52,7 +52,7 @@
       </li>
       <li class="nav-item">
         <a href="#" class="nav-link nav-icon">
-          <img src="/docs/{{ $.Site.Params.docs_version }}/assets/img/navbar-contact.png" role="img" alt="User" loading="lazy" aria-hidden="true">
+          <img src="/docs/{{ $.Site.Params.docs_version }}/assets/img/navbar-contact.png" alt="" loading="lazy" aria-hidden="true">
```

Comment:
This is the only place where we have a `navbar-contact.png` combined with an `alt=""`, but sill also an `aria-hidden="true"`.

```suggestion
          <img src="/docs/{{ $.Site.Params.docs_version }}/assets/img/navbar-contact.png" alt="" loading="lazy">
```